### DISCUSSION TITLE
Push-to-talk

### DISCUSSION THREAD

**Discussion by artemka2281** [stoatchat/discussions#794] [Feature Suggestions]
Hello, could you please add Push-to-talk? I think it will be convenient for many people)
UPD:Guys, I noticed this thing: when you send emoticons, you cannot send several at once, you have to open this window every time. To be honest, it's not very convenient

**Comment by soufied** [comment]
yeah we need it

**Comment by R1werCrack** [comment]
100% we need

**Comment by Cookie0fPower** [comment]
This needs more attention.

I hope to see PTT (Push-to-Talk) implemented in the near future. Especially for Users with larger communities.

**Reply by ThomasJRyan** [reply]
@Cookie0fPower it doesn't do anyone any good (other than me I guess so I can close them) to revive several old discussions that all target the same topic. Please refrain in the future

**Comment by Dhenyson** [comment]
How does a voice communication app not have "push-to-talk"? It's a basic feature; as a dev, I'd implement that before even building a text chat, haha.

**Reply by IAmTomahawkx** [reply]
https://x.com/FFmpeg/status/1762805900035686805

Also, we are not a voice communication app. Our platform is primarily text-based.
